src/gui/qjacktrip.cpp

```diff
@@ -832,10 +876,6 @@ void QJackTrip::loadSettings()
     m_ui->backendComboBox->setCurrentIndex(settings.value("Backend", 0).toInt());
     m_ui->sampleRateComboBox->setCurrentText(settings.value("SampleRate", "48000").toString());
     m_ui->bufferSizeComboBox->setCurrentText(settings.value("BufferSize", "128").toString());
-    if (m_ui->backendComboBox->currentIndex() == 0) {
```

Comment:
Yeah, I understand this shouldn't be needed, but when I first implemented it, for some reason I observed not having these boxes disabled on load. Maybe it was a build error or something. Works fine in this PR.